terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/test/java/org/terasoluna/gfw/common/validator/constraints/ByteMaxTest.java

```diff
@@ -143,6 +155,97 @@ public void testAnnotateUnexpectedType() throws Throwable {
         validator.validate(form, UnexpectedType.class);
     }
 
+    /**
+     * all values in the collection are valid.
+     * @throws Throwable
```

Comment:
- [ ] Please add description of `@throws`. Or is ` throws Throwable` necessary?

Reply:
9847a8610ea9c0bface8d17a775ce2f339328e7b

No, throws `Throwable` is need not necessary . I removed .